addons/sourcemod/scripting/gokz-hud/info_panel.sp

```diff
@@ -27,14 +27,7 @@ bool IsDrawingInfoPanel(int client)
 
 void OnPlayerRunCmdPost_InfoPanel(int client, int cmdnum, HUDInfo info)
 {
-	int updateSpeed = 10;
-	if (gB_FastUpdateRate[client])
-	{
-		// The hint text panel update speed depends on the client ping.
-		// To optimize resource usage, we scale the update speed with it.
-		// The fastest speed the client can get is around once every 2 ticks.
-		updateSpeed = IntMax(1, RoundToFloor(GetClientAvgLatency(client, NetFlow_Outgoing) / GetTickInterval()));
-	}
+	int updateSpeed = gB_FastUpdateRate[client] ? 3 : 6;
```

Comment:
Why did the slow update speed change from 10 to 6?

Reply:
To make it in line with gokz's other speed panel, now that they behave identically regardless of latency. Could revert if you want though, of course.